src/app/store.ts

```diff
@@ -1,6 +1,8 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { currentTodoSlice, todosSlice } from '../features';
+import { filterSlice } from '../features';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(todosSlice, currentTodoSlice, filterSlice);
```

Comment:
The `combineSlices` function is not a valid function in Redux Toolkit. You should use `combineReducers` instead to combine the slices: `const rootReducer = combineReducers({ todos: todosSlice.reducer, currentTodo: currentTodoSlice.reducer, filter: filterSlice.reducer });`